### ISSUE TITLE
Search input modal allows user to tab away from it without explicitly dismissing it

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#83]
x-ref to user's feedback on PST https://github.com/pydata/pydata-sphinx-theme/issues/1403
